spec/views/posts/posts_index_spec.rb

```diff
@@ -35,10 +35,28 @@
       expect(page).to have_content(@comment.text)
     end
 
```

Comment:
- [ ] These tests are the same as the ones you created in the `users_index_spec.rb` kindly make sure to not repeat the same tests in different files, instead you should keep these tests and modify the `users_index_spec.rb` to test the `Users` index page